subsys/bluetooth/host/monitor.c

```diff
@@ -381,22 +412,30 @@ static int bt_monitor_init(const struct device *d)
 {
 	ARG_UNUSED(d);
 
+#if defined(CONFIG_BT_DEBUG_MONITOR_RTT)
+	/* RTT up buffer */
+	static uint8_t rtt_buf[RTT_BUF_SIZE];
```

Comment:
Don't we now have 2 static `rtt_buf` in the code? Here and on line 103. 

Seems like something that can cause issues/confusion

Reply:
The scope is different, as one was declared in function and another was declared outside of it.
Anyway, I've changed the name of one of those as you probably noticed